File: Python_parte1/lista_1/ex3.py
print("Código do produto \n 100 \n 101 \n 112 \n 124 ")
print("Nome \n Cachorro quente \n Refrigerante \n Pão na chapa \n Pão de queijo ")
print("Preço \n R$ 6,00 \n R$ 4,00 \n RS4,50 \n R$ 2,50 ")

codigo = int(input("Insira o código do produto que você deseja comprar: "))

if codigo == 100:
    nome = "Cachorro quente"
    valor = 6.00
    print(f"Você escolheu o {nome}, o valor da compra ficará por {valor} reais")
elif codigo == 101:
    nome = "Refrigerante"
    valor = 4.00
    print(f"Você escolheu o {nome}, o valor da compra ficará por {valor} reais")
elif codigo == 112:
    nome = "Pão na chapa"
    valor = 4.50
    print(f"Você escolheu o {nome}, o valor da compra ficará por {valor} reais")
elif codigo == 124:
    nome = "Pão de queijo"
    valor = 2.50
    print(f"Você escolheu o {nome}, o valor da compra ficará por {valor} reais")
else:
    print("Insira um código válido!!!")

File: C_parte2/lista_4/ex3.c
#include <stdio.h>

int main(void)
{

    int codigo;

    printf("Insira o codigo do produto que voce deseja comprar: ");
    scanf("%d", &codigo);

    switch (codigo)
    {
    case 100:
        printf("Voce selecionou o Cachorro quente, ficara R$ 6,00 reais\n");
        break;
    case 101:
        printf("Voce selecionou o Refrigerante, ficara R$ 4,00 reais\n");
        break;
    case 112:
        printf("Voce selecionou o Pao na chapa, ficara R$ 4,50 reais\n");
        break;
    case 124:
        printf("Voce selecionou o Pao de queijo, ficara R$ 2,50 reais\n");
        break;
    default:
        printf("Opcao invalida!!!\n");
        break;
    }


    return 0;
}